omnibus/config/chefdk_overrides.rb

```diff
@@ -0,0 +1,23 @@
+override :berkshelf, :version => "v4.3.1"
```

Comment:
It *is* intended for `override` in the project file. It's loaded directly by the project file :) This is just a cheap way for us to control and extract the list of gems.